Ontology/Asset/Component/HVACCoolingMethod.json

```diff
@@ -0,0 +1,78 @@
+{
+  "@id": "dtmi:digitaltwins:rec_3_3:asset:HVACCoolingMethod;1",
+  "@type": "Interface",
+  "displayName": {
+    "en": "Cooling Method"
+  },
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "type",
+      "displayName": {
+        "en": "Type"
+      },
+      "writable": true,
+      "schema": {
+        "@type": "Enum",
+        "valueSchema": "string",
+        "enumValues": [
+          {
+            "name": "DX",
+            "displayName": {
+              "en": "DX"
+            },
+            "enumValue": "DX"
+          },
+          {
+            "name": "ChilledWater",
+            "displayName": {
+              "en": "Chilled Water"
+            },
+            "enumValue": "chilledWater"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Property",
+      "name": "rows",
```

Comment:
This name is rather broad, and the subsequent comment ("Coil row configuration") is very narrow. Also, the name indicates a numeric value (consistent with the selected schema) but the comment indicates a (potentially complex) configuration value. Could better consistency between naming and comment, in terms of schema and granularity, be achieved?

Reply:
The `quantity of rows` is a configuration option when selecting a coil. I'm ok changing this to something that better indicates that this is a quantity of rows such as coilRows or coilRowQuantity?

Here is a link to the manufacturer [Trane's typical Air Handling Units](https://www.trane.com/content/dam/Trane/Commercial/global/products-systems/equipment/air-handling/catalog/CLCH-PRC024J-EN_03122020.PDF).
(Pgs. 6, 18)